test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1429,16 +1429,23 @@ public void TensorFlowImageClassification(ImageClassificationTrainer.Architectur
         [TensorFlowFact]
         public void TensorFlowImageClassificationWithExponentialLRScheduling()
         {
-            TensorFlowImageClassificationWithLRScheduling(new ExponentialLRDecay());
+            TensorFlowImageClassificationWithLRScheduling(new ExponentialLRDecay(), 50);
         }
 
-        [Fact(Skip ="Very unstable tests, causing many build failures.")]
+        [TensorFlowFact]
         public void TensorFlowImageClassificationWithPolynomialLRScheduling()
         {
-            TensorFlowImageClassificationWithLRScheduling(new PolynomialLRDecay());
+
+            /*
+             * Due to an issue with Nix based os performance is not as good,
+             * as such increase the number of epochs to produce a better model.
+             */
```

Comment:
I think this is because of some images that seem to have different pixel format on nix than windows, can we please try changing the dataset by subsampling another dataset like cats vs dogs?

Reply:
Discussed offline will look into this and potentially replace the dataset in a separate PR, or potentially replace just the bad images if possible.

---
In reply to: [344365787](https://github.com/dotnet/machinelearning/pull/4446#discussion_r344365787) [](ancestors = 344365787)

Reply:
thanks